apps/dashboard/src/main/java/com/akto/action/testing/StartTestAction.java

```diff
@@ -531,17 +530,23 @@ public String fetchTestingRunResults() {
                 case VULNERABLE:
                     testingRunResultFilters.add(Filters.eq(TestingRunResult.VULNERABLE, true));
```

Comment:
Add filters.nin in vulnerable otherwise it will show SKIPPED_EXEC_API_REQUEST_FAILED these also

Reply:
check line 534 535, 
did that exactly

Reply:
also, 
for this case, no test will be vulnerable, as api request failed,